### PR TITLE
feat:cartpage and wishlist page added

### PR DESCRIPTION
**Pull request by ArpitPatni**
Please review my Cart page and wishlist page.
Demo Link:
https://shoeestore.netlify.app/

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -2,6 +2,8 @@ import "./App.css";
 import { Routes, Route } from "react-router-dom";
 import HomePage from "./Pages/Homepage/HomePage";
 import Products from "./Pages/Products/Products";
+import MyCart from "./Pages/MyCart/MyCart";
+import MyWishlist from "./Pages/MyWishlist/MyWishlist";
```
Comment (pnchinmay) [inline src/App.js:6]: Barrel imports are recommended

**Review by pnchinmay — APPROVED** [review]

File: src/App.js (hunk 1 above)
Comment (ArpitPatni) [reply src/App.js:6]: For some reason auto imports doesn't work in my VS code this is why I wasn't able to use barrel imports.
Thank you.I will keep this in mind for the next projects.

File: src/Pages/MyCart/MyCart.css [hunk 2]
```diff
@@ -0,0 +1,199 @@
+@import url("https://fonts.googleapis.com/css2?family=Montserrat&display=swap");
+
+:root {
+  --body-background-color: rgba(128, 128, 128, 0.322);
+  --background-color: #2874f0;
+  --white-color: white;
+  --black-color: black;
+  --light-gray: lightgray;
+  --gray-color: gray;
```
Comment (raieegulhane) [inline src/Pages/MyCart/MyCart.css:9]: hex/rgb might be preferable here

File: src/Pages/MyCart/MyCart.jsx [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+                                            value={prod.qty}
+                                            onChange={(e) =>
+                                                productsDispatch({
+                                                    type: "CHANGE_IN_QTY",
+                                                    payload: { id: prod.id, qty: e.target.value },
+                                                })
+                                            }
+                                        >
+                                            {[...Array(prod.inStock).keys()].map((x) => (
+                                                <option key={x + 1}>{x + 1}</option>
```
Comment (raieegulhane) [inline src/Pages/MyCart/MyCart.jsx]: use proper name instead of x

**Review by raieegulhane — COMMENTED** [review]
Small suggestions in comments. Rest good to merge.

**Review by rahul31agrawal — APPROVED** [review]
Apart from mentioned suggestion,
Looks good to merge

File: src/Pages/MyCart/MyCart.css (hunk 2 above)
Comment (ArpitPatni) [reply src/Pages/MyCart/MyCart.css:9]: Thank you for the suggestion!

File: src/Pages/MyCart/MyCart.jsx (hunk 3 above)
Comment (ArpitPatni) [reply src/Pages/MyCart/MyCart.jsx]: thank you for the suggestion!

**Pull request merged**